qiskit/ignis/experiments/calibration/cal_base_experiment.py

```diff
@@ -19,16 +19,17 @@
 from qiskit.providers import BaseBackend, BaseJob
 
 from qiskit.ignis.experiments.base import Experiment
-from qiskit.ignis.experiments.calibration import cal_base_generator, cal_base_fitter
+from qiskit.ignis.experiments.calibration import Calibration1DAnalysis
 from qiskit.ignis.experiments.calibration.exceptions import CalExpError
 from qiskit.ignis.experiments.calibration.workflow import AnalysisWorkFlow
+from qiskit.ignis.experiments.base import Generator
 
 
 class BaseCalibrationExperiment(Experiment):
     """An experiment class for a calibration experiment."""
     def __init__(self,
-                 generator: Optional[cal_base_generator.BaseCalibrationGenerator] = None,
-                 analysis: Optional[cal_base_fitter.BaseCalibrationAnalysis] = None,
+                 generator: Optional[Generator] = None,
+                 analysis: Optional[Calibration1DAnalysis] = None,
```

Comment:
```suggestion
                 analysis: Optional[Analysis] = None,
```
because we may use another fitter which directly derived from superclass.